bloom-housing/bloom
# fix: script runner for new reserved community type

This PR addresses #[853](https://github.com/metrotranscom/doorway/issues/853)


## Description

Pulls in the create reserved community type script runner and adds the translations for the School Employee community type.

Note: the script runner in HBA was specific to a certain community type. This PR makes it more general so that we can use the same script for future community types

Note: Translations have not been confirmed by product

## How Can This Be Tested/Reviewed?

1. Reseed the backend and start it up
2. Log in as a admin user (either through the partner UI or directly in the swagger doc)
3. In the swagger doc http://localhost:3100/api#/scriptRunner/createNewReservedCommunityType run the script using the following data:
  - id : jurisdiction id (need to get it from the db)
  - name: "schoolEmployee"
  - description: "Employee of the school district"
4. The new community type should be in the database and also selectable in the partner UI

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
